### PR TITLE
Qualified constructors are not valid identifiers for operator aliases

### PR DESCRIPTION
**Pull request by LiamGoodacre**
For https://github.com/purescript/purescript/issues/2014

### CODE REVIEW

**Comment by garyb** [comment]
That was quick :wink:

Looks good to me.

File: src/Language/PureScript/Parser/Declarations.hs [hunk 1]
```diff
@@ -126,7 +126,7 @@ parseFixityDeclaration = do
   name <- symbol
   return $ FixityDeclaration fixity name alias
   where
-  aliased = (Left <$> parseQualified (Ident <$> identifier))
+  aliased = P.try (Left <$> parseQualified (Ident <$> identifier))
         <|> (Right <$> parseQualified (ProperName <$> uname))
```
Comment (LiamGoodacre) [inline src/Language/PureScript/Parser/Declarations.hs]: Also `ProperName <$> uname` is the same as `properName` from `Common` - should I switch it to that?

Comment (garyb) [inline src/Language/PureScript/Parser/Declarations.hs]: Yeah, I guess so! I probably just copied the `Ident` case when adding constructors. We intentionally don't use the `Common` one there as `parseIdent` will accept `Op` or `Ident`.

Comment (LiamGoodacre) [inline src/Language/PureScript/Parser/Declarations.hs]: Yeah I noticed the difference to `parseIdent` :]

**Pull request merged**